source/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,156 @@
+// Variables
+
+$light:  #febbbb;
+$medium: #fe9090;
+$dark:   #ff5c5c;
+$dark-v1: #c84848;
+$text-color: #999;
+$cta: #00c5a7;
+$card-font-small: 15px;
+
+
+// Global Styles
+/*#press {
+  background: $light;
+  background: linear-gradient(135deg, $light 0%,$medium 45%,$dark 100%);
+  font-size: 10px;
+  padding: 2em;
+}
+.full-screen {
+  height: 100vw;
+  position: relative;
+}*/
+
+
+// General content
+.title {
+  text-align: center;
+  font-weight: 100;
+}
+.divider {
+  border-bottom: 1px solid $light;
+}
+
+
+
+/* Globals for cards */
+
+// Cards
+.cards {
+  background: #f2f2f2;
+  border-radius: 3px;
+  cursor: pointer;
+  height: 17em;
+  padding: 1.5em;
+  position: relative;
+  transition: height 0.4s, box-shadow 0.5s;
+
+  &:hover {
+    box-shadow: 0 0 2em #999;
+    height: 38em;
+    transition: height 0.3s ease, box-shadow 0.5s ease;
+
+    .divider {
+      opacity: 0.5;
+    }
+
+    .info {
+      transition: top 0.5s ease, opacity 1s ease;
+      visibility: visible;
+      top: 1%;
+      opacity: 1;
+      .divider {
+        /*//opacity: 1;*/
+      }
+
+    .font-small {
+      font-size: $card-font-small;
+    }
+  }
+
+    .icon-center {
+      font-size: 2em;
+      left: 45%;
+      transition: font-size .3s, left .3s;
+    }
+    .icon-right {
+      font-size: 2em;
+      right: 45%;
+      transition: font-size .3s, right .3s;
+    }
+  }
+
+
+  .info {
```

Comment:
Was thinking about doing `.img-center {}`, but was afraid of overriding  the `img {}` selector or creating a new unnecessary one. Should I change them to `img-center` and `img-right` right respectively?